docs/dev/libvirt-howto.md

```diff
@@ -22,7 +22,7 @@ If it is missing, try some of the ideas [here][kvm-install].
 On Fedora, CentOS/RHEL:
 
 ```sh
-sudo yum install libvirt libvirt-devel
+sudo yum install libvirt libvirt-devel qemu-kvm
```

Comment:
Last touch was #514.  I'd really rather get something backed by CI, like [this][1], which is entering CI via openshift/release#2143.  Based on that (it still hasn't landed), we'd also want libvirt-client, libvirt-daemon-kvm, and bind-utils.    Thoughts?

[1]: https://github.com/ironcladlou/openshift4-libvirt-gcp/blob/794baf7fb6e2a8261ec260b6dfa9de06a0136a81/provision.sh#L34

Reply:
When I ran through this on a clean RHEL install the above was enough that I got `bind-utils` for "free" (may have been part of the minimal config?) but yes `libvirt-daemon-kvm` is needed as well.

Reply:
What about `libvirt-client`?  We may not need it for the build, but we have a few `virsh` calls in these docs.  Or does it get pulled in by another package as well?

Reply:
It gets pulled in by the above.